### PR TITLE
Exposing the deployed Grafana in the Loki tutorial

### PR DESCRIPTION
**Pull request by Teneroy**
**Description**

Changes proposed in this pull request:

- Added apirule file for grafana;
- Added grafana exposure section to README;

**Related issue(s)**

https://github.com/kyma-project/kyma/issues/15828

### CODE REVIEW

**Review by chrkl — DISMISSED** [review]

File: loki/README.md [hunk 1]
```diff
@@ -101,8 +101,21 @@ When the status of the applied LogPipeline resource turns into `Running`, the un
      ```bash
      kubectl -n ${KYMA_LOKI_EXAMPLE_NS} port-forward svc/${HELM_RELEASE_NAME}-grafana 3000:80
      ```
-
-  3. Open Grafana in your browser under `http://localhost:3000` and log in with user admin and the password taken from the previous Helm command.
+     
+     Open Grafana in your browser under `http://localhost:3000` and log in with user admin and the password taken from the previous Helm command.
+  
+  3. To expose Grafana using the Kyma API Gateway, download the api rule file and replace the `{release-name}` to the name of Helm release:
```
Comment (NHingerl) [inline loki/README.md:107]:
```suggestion
  3. To expose Grafana using the Kyma API Gateway, download the APIRule file and replace the `{release-name}` variable with the name of Helm release:
```

File: loki/README.md [hunk 2]
```diff
@@ -101,8 +101,21 @@ When the status of the applied LogPipeline resource turns into `Running`, the un
...
+     ```bash
+     curl https://raw.githubusercontent.com/kyma-project/examples/main/loki/apirule.yaml -o apirule.yaml
+     ```
+     Create an APIRule:
```
Comment (NHingerl) [inline loki/README.md:111]: It's an instruction, why not numbered?

```suggestion
4. Create an APIRule:
```

File: loki/README.md [hunk 3]
```diff
@@ -101,8 +101,21 @@ When the status of the applied LogPipeline resource turns into `Running`, the un
...
+     ```bash
+     kubectl -n ${KYMA_LOKI_EXAMPLE_NS} apply -f apirule.yaml 
+     ```
+     Get the public URL of your Kiali server:
```
Comment (NHingerl) [inline loki/README.md:115]:
```suggestion
5. Get the public URL of your Kiali server:
```

**Review by grego952 — DISMISSED** [review]

**Review by chrkl — DISMISSED** [review]

**Review by chrkl — APPROVED** [review]

File: loki/README.md [hunk 4]
```diff
@@ -101,9 +101,27 @@ When the status of the applied LogPipeline resource turns into `Running`, the un
...
+4. Create an APIRule:
+     ```bash
+     kubectl -n ${KYMA_LOKI_EXAMPLE_NS} apply -f apirule.yaml 
+     ```
+5. Get the public URL of your Kiali server:
+     ```bash
+     kubectl -n ${KYMA_LOKI_EXAMPLE_NS} get vs -l apirule.gateway.kyma-project.io/v1beta1=grafana.${KYMA_LOKI_EXAMPLE_NS} -ojsonpath='{.items[*].spec.hosts[*]}'
+     ```
+
+  4. To add the link to exposed Grafana to kyma dashboard, we should edit the `dashboard-configmap.yaml` and change the `{grafana-link}` to the text you retrieved from the previous step. You can change `label` field if you wish to change the name of the tab, but also the `category` tab if you wish to move it to another category. After, apply this `ConfigMap`, go to the busola dashboard, and you will see your newely exposed Grafana under Observability section. If you already have some `busola-config`, you should merge it with existing one:
```
Comment (grego952) [inline loki/README.md:120]: This point 4 looks like another set of instructions, maybe we can use an ordered list, like in previous points? For example:
###Add the link of the exposed Grafana to Kyma Dashboard
1. Edit `dashboard-configmap.yaml` and change `{GRAFANA_LINK}` to the text you retrieved in the previous step.

>**NOTE:** You can change the label field to change the name of the tab, and the category tab if you wish to move it to another category.

3. Apply this ConfigMap, and go to Kyma Dashboard. You should see your newly exposed Grafana under the Observability section.
4. If you already have `busola-config`, merge it with the existing one:

File: loki/README.md [hunk 5]
```diff
@@ -101,9 +101,27 @@ When the status of the applied LogPipeline resource turns into `Running`, the un
...
   
+  3. To expose Grafana using the Kyma API Gateway, download the APIRule file and replace the `{release-name}` variable with the name of Helm release:
+     ```bash
+     curl https://raw.githubusercontent.com/kyma-project/examples/main/loki/apirule.yaml -o apirule.yaml
+     ```
+4. Create an APIRule:
+     ```bash
+     kubectl -n ${KYMA_LOKI_EXAMPLE_NS} apply -f apirule.yaml 
+     ```
+5. Get the public URL of your Kiali server:
```
Comment (a-thaler) [inline loki/README.md:115]:
```suggestion
5. Get the public URL of your Loki instance:
```

File: loki/README.md [hunk 6]
```diff
@@ -90,20 +90,48 @@ When the status of the applied LogPipeline resource turns into `Running`, the un
...
+     ```bash
+     kubectl -n ${KYMA_LOKI_EXAMPLE_NS} get vs -l apirule.gateway.kyma-project.io/v1beta1=grafana.${KYMA_LOKI_EXAMPLE_NS} -ojsonpath='{.items[*].spec.hosts[*]}'
+     ```
+
+  ### Add a Link for Grafana to the Kyma Dashboard
+  1. Download the `dashboard-configmap.yaml` file and change `{GRAFANA_LINK}` to the text you retrieved in the previous step.
+     ```bash
+       curl https://raw.githubusercontent.com/kyma-project/examples/main/loki/dashboard-configmap.yaml -o dashboard-configmap.yaml
+     ```
+     NOTE: You can change the label field to change the name of the tab, and the category tab if you wish to move it to another category.
```
Comment (grego952) [inline loki/README.md:128]:
```suggestion
     >NOTE: You can change the label field to change the name of the tab, and the category tab if you wish to move it to another category.
```

**Review by grego952 — APPROVED** [review]

**Review by lindnerby — APPROVED** [review]

**Pull request merged**